tests/events/18-suite-event-mail.t

```diff
@@ -16,6 +16,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
 # Test event mail.
+CYLC_TEST_IS_GENERIC=false
```

Comment:
It is "platform specific" in the specific (and admittedly dubious) sense that I didn't figure out how to get a working 'mail' command on the Travis CI platform.  I suppose I should have put the mail tests in the new CYLC_TEST_SKIP list rather than PLATFORM.  (unless you can advise on use of "mock mail server"?)